src/UglyToad.PdfPig/Content/IPdfImage.cs

```diff
@@ -89,10 +95,10 @@ public interface IPdfImage
         ColorSpaceDetails? ColorSpaceDetails { get; }
 
         /// <summary>
-        /// Get the decoded bytes of the image if applicable. For JPEG images and some other types the
-        /// <see cref="RawBytes"/> should be used directly.
+        /// Get the decoded memory of the image if applicable. For JPEG images and some other types the
+        /// <see cref="RawMemory"/> should be used directly.
         /// </summary>
-        bool TryGetBytes([NotNullWhen(true)] out IReadOnlyList<byte>? bytes);
+        bool TryGetMemory(out ReadOnlyMemory<byte> memory);
```

Comment:
can you rename `TryGetMemory(...)` into `TryGetBytesAsMemory(...)`? I think it makes the method function clearer